src/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Store from './redux/Store';
+import './index.css';
+import GetAllManufacturers from './Components/getAllManufacturers';
+import GetByModel from './Components/getByModel';
+import GetByModelForMake from './Components/getByModelForMake';
+import GetByName from './Components/getByName';
+import GetByNameAndYear from './Components/getByNameAndYear';
+import GetByTypesForMake from './Components/getByTypesForMake';
+import GetByYear from './Components/getByYear';
+import GetCanadians from './Components/getCanadians';
+import GetMakesByName from './Components/getMakesByName';
+import Home from './Home';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <Provider store={Store}>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="GetAllManufacturers" element={<GetAllManufacturers />} />
+        <Route path="GetByModel" element={<GetByModel />} />
+        <Route path="GetByModelForMakel" element={<GetByModelForMake />} />
+        <Route path="GetByName" element={<GetByName />} />
+        <Route path="GetByNameAndYear" element={<GetByNameAndYear />} />
```

Comment:
- [ ] [OPTIONAL] I don't believe it is ideal to have all of these routings and imports in your `index.js` file. It is better, cleaner and more maintainable to have an `App.js` component that handles all of this, then you simply render that component in the index.js file 😎